loops_functions.py

```diff
@@ -0,0 +1,35 @@
+
+def atm():
+    current_balance = 500000
+    min = 50
+    max = 5000
+
+    while True:
+        print("current balance:", current_balance)
+        
+        user_input = input("Do you want to withdraw money? (yes/no): ")
+
+        if user_input == 'no':
+            print("Have a nice day!")
+            break
+
+        elif user_input == 'yes':
+            amount= int(input("enter the amount: "))
```

Comment:
this is logically correct, but can be catastrophic in future, this is because the user can input something that isnt an integer.